src/main/resources/arrebol-k8s.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I see now. Why a json file instead of a conf file? This structure is more complex then a simple conf file, don't you think? Just a note, though.

Reply:
I followed the same structure used for `arrebol.json`, do you think it is better to change that?